tests/infra/consortium.py

```diff
@@ -7,21 +7,62 @@
 import json
 import random
 import re
-import uuid
+import subprocess
+import tempfile
 import infra.network
 import infra.proc
 import infra.checker
 import infra.node
 import infra.crypto
 import infra.member
 from ccf.ledger import NodeStatus
-import ccf.proposal_generator
+import ccf.ballot_builder
+import ccf.bundle_js_app
 import ccf.ledger
 from infra.proposal import ProposalState
 
+from cryptography import x509
+import cryptography.hazmat.backends as crypto_backends
+
 from loguru import logger as LOG
 
 
+def generate_proposal(proposal_name, **kwargs):
```

Comment:
I am very confused by the need for this wrapper for build_proposal.sh. Why do we need it at all? It seems to that the benefit of using jinja templates should be to replace our instances of this with a very short call to their instantiation.

Reply:
`build_proposal.sh` is not using a Jinja template. I should remove the Jinja template for constructing proposals. By the time you've constructed a structured input that you could pass to a template, you've already constructed a proposal. We can drop the invocation of `build_proposal.sh` here and just use Python.json to produce proposals. `ballot_builder` is just the a Python wrapper for instantiating a Jinja template.